examples/edge_bundling.ipynb

```diff
@@ -0,0 +1,524 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "def generate_random_graph(nodes, edges):\n",
+    "    ndf = generate_nodes(nodes)\n",
+    "    edf = generate_edges(edges, ndf)\n",
+    "    return ndf, edf"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "nodes_df, edges_df = generate_random_graph(10000, 50000)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "%time df = bundle(nodes_df, edges_df, 0.05, 0.7)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "render_lines(df, width=2000, height=2000, cmap=viridis, bgcolor='black')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true
+   },
+   "source": [
+    "### Edge bundling with star graph and circular layout"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import networkx as nx"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "graph = nx.star_graph(100000)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "layout = nx.circular_layout(graph)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "data = []\n",
+    "for node in graph.nodes_iter():\n",
+    "    x, y = layout[node]\n",
+    "    data.append([node, x, y])\n",
+    "nodes_df = pd.DataFrame(data, columns=['id', 'x', 'y'])\n",
+    "nodes_df.set_index('id', inplace=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "edges_df = pd.DataFrame(graph.edges_iter(), columns=['source', 'target'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "%time df = bundle(nodes_df, edges_df, 0.05, 0.7)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "render_lines(df, width=2000, height=2000, cmap=inferno, bgcolor='black')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Decrease decay for edge bundling"
+   ]
```

Comment:
Can you add a sentence or two about each parameter examined?  E.g. the decay value determines how much the edges are bundled, with lower values giving more separate bundles, and higher values giving fewer, larger bundles? It's tricky because the initial bandwidth also seems to be describable the same way.